content/covid19/_index.md

```diff
@@ -4,11 +4,75 @@ linkTitle: COVID-19
+2. We make all COVID-19 indicators freely available through a
+  [public API](https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html),
+  which is updated with new data daily. We also provide associated
+  [R](https://cmu-delphi.github.io/covidcast/covidcastR/) and
+  [Python](https://cmu-delphi.github.io/covidcast/covidcast-py/html/) packages, 
+  and a [dashboard]({{< relref "covidcast" >}}) to visualize a core set of
+  indicators. 
 
-An overview of our work and data was [published in the *Proceedings of the National Academy of Sciences*](https://doi.org/10.1073/pnas.2111452118). We have selected a small number of indicators to showcase in [COVIDcast]({{< relref "covidcast" >}}), a visualization system that helps place pandemic activity in geographic and temporal context. It includes a notion of correlation across time, for example, a spike in cases is often followed by a spike in hospital admissions some number of days later.
+3. Some of the underlying data sources (on which these indicators are built) 
+  would not exist or be publicly available without Delphi's efforts. These
+  include a massive national daily survey called the [COVID-19 Trends and
+  Impact Survey (CTIS)]({{< relref "covid19/ctis" >}}) that we're running in
+  partnership with Facebook, which has been taken by over 25 million Americans
+  since April 2020; and an enormous database of de-identified medical insurance
+  claims, covering more than 35% of the U.S. population, made possible through
+  health system partners including Change Healthcare. 
+
+4. We support and advise the U.S. CDC (in collaboration with the
+  [Reich Lab](https://reichlab.io/)) on short-term COVID forecasting, which 
+  includes creating and evaluating an ensemble forecast out of the models
+  submitted to the [COVID-19 Forecast Hub](https://covid19forecasthub.org/),
+  serving as the basis for the [CDC's official forecast
+  communications](https://www.cdc.gov/coronavirus/2019-ncov/covid-data/forecasting-us.html).
+
+## See Also
+
+- [COVIDcast dashboard]({{< relref "covidcast" >}})
+- [CTIS dashboard]({{< relref "covidcast/survey-results" >}})
+- [Export data tool]({{< relref "covidcast/export" >}})
+- [API documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html)
+- [R package](https://cmu-delphi.github.io/covidcast/covidcastR/)
+- [Python package](https://cmu-delphi.github.io/covidcast/covidcast-py/html/)
+- [GitHub repo](https://www.github.com/cmu-delphi/covidcast/) (see also
+  [related repos](https://github.com/cmu-delphi/covidcast/#related-repos))
+  
+## Research Articles
+
+A summary of some key parts of COVIDcast can be found in the December 2021 PNAS
```

Comment:
```suggestion
A summary of some key parts of COVIDcast can be found in the December 2021 *Proceedings of the National Academy of Sciences*
```
since a lot of people may not know the acronym by sight, even if they do know of the journal